_posts/2022-12-04-OurFourSins.md

```diff
@@ -0,0 +1,6 @@
+---
+Title: "Our Four Sins"
+Date: 2022-12-04
+Author: heyysiri
+---
+<img width="333" alt="2022-12-04" src="https://user-images.githubusercontent.com/113199947/205488008-76758a64-1ab9-4364-bc91-9ae5e066cb17.png">
```

Comment:
change img width to 666. right now its too small.

Reply:
@heyysiri The full content should not be embedded in an image ( but your blog may contain images ). It will make your post's aesthetics look bad.